source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,57 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template<typename Policy, typename KeyIt, typename ValueIt,
+        typename Comparator = std::less<typename std::iterator_traits<InputIt>::value_type>>
+    void
+    sort_by_key(Policy&& policy, KeyIt keys_first, KeyIt keys_last,
+        ValueIt values_first,
+        Comparator comp = std::less<typename std::iterator_traits<InputIt>::value_type>());
+
+``oneapi::dpl::sort_by_key`` sorts the sequence of keys ``[keys_first, keys_last)``
+and simultaneously permutes associated values at the same positions in the range
+``[values_first, values_first + std::distance(keys_first, keys_last))``
+to match the order of the sorted keys.
```

Comment:
@akukanov, what do you think about adding "in the range" to be more explicit?

```suggestion
``oneapi::dpl::sort_by_key`` sorts the sequence of keys in the range ``[keys_first, keys_last)``
and simultaneously permutes associated values at the same positions in the range
``[values_first, values_first + std::distance(keys_first, keys_last))``
to match the order of the sorted keys.
```

Reply:
Actually, I removed something similar because of a confusion for "a/the sequence in the range". I think of ``[keys_first, keys_last)`` as **the** sequence. In other words, "the sequence" and "the range" are mostly interchangeable for me.